bin/phylo_dist/time_calibrate_species_tree.R

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env Rscript
+
+# Time-calibrate a consensus species tree using a reference time-calibrated tree
+# This script uses geiger::congruify.phylo() to transfer time-calibration from
+# a reference tree to a target consensus tree.
+
+# Load required libraries
+library(ape)
+library(geiger)
+
+# Parse command line arguments
+args <- commandArgs(trailingOnly = TRUE)
+
+if (length(args) < 4) {
+  stop(paste(
+    "Usage: time_calibrate_species_tree.R <consensus_tree> <reference_tree> <output_tree> <calibration_method>",
+    "\n  consensus_tree: Path to consensus species tree (Newick format)",
```

Comment:
I might suggest switching from "consensus tree" to "species tree" in any references/parameter/variable names for clarity, since this will always be the non-time-calibrated species tree, and "consensus" is a bit confusing.